src/datamodel_code_generator/parser/jsonschema.py

```diff
@@ -303,7 +304,7 @@ def is_object(self) -> bool:
 
     @cached_property
     def is_array(self) -> bool:
-        return self.items is not None or self.type == "array"
+        return self.items is not None or self.prefixItems is not None or self.type == "array"
```

Comment:
I see in the jsonschema documentation [^1] you can set `items: false`.

- [ ] Does this work in that case?
- [ ] Could you please add a test for that?

[^1]: http://json-schema.org/understanding-json-schema/reference/array#additionalitems